### PR TITLE
PolicySet Zettaset Xcrypt

### PR DESCRIPTION
**Pull request by mahesh-zetta**

### CODE REVIEW

File: policygenerator/policy-sets/community/zts-xcrypt/policyGenerator.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - path: input/operator-dh-v1-local.yaml
+  remediationAction: enforce 
+- name: policy-zts-xcrypt-operator-executor-deployment  
+  disabled: false
+  manifests:
+    - path: input/operator-executor.yaml
+  remediationAction: enforce 
+policySets:
+- name: policyset-xcrypt
+  description: this is a zts-xcrypt
```
Comment (gparvin) [inline policygenerator/policy-sets/community/zts-xcrypt/policyGenerator.yaml:42]: Can you have a more detailed description?

File: policygenerator/policy-sets/community/zts-xcrypt/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+zts-csi-controller-7576b65bf7-6gs5v    4/4     Running   0          21m
+zts-csi-node-84wlr                     3/3     Running   0          21m
+zts-csi-node-bz2sk                     3/3     Running   0          21m
+zts-masterset-ca-429bz                 1/1     Running   0          21m
+zts-masterset-hm-r4gkt                 1/1     Running   0          21m
+zts-masterset-hm-v7ss8                 1/1     Running   0          21m
+zts-masterset-kmip-fsdkr               1/1     Running   0          21m
+zts-masterset-ls-7nrn5                 1/1     Running   0          21m
+zts-xcrypt-operator-68f4d949c6-mzxqs   1/1     Running   0          21m
+[docker@localhost managed]$
```
Comment (gparvin) [inline policygenerator/policy-sets/community/zts-xcrypt/README.md:47]: This content should be enclosed as a code block.

File: policygenerator/policy-sets/community/zts-xcrypt/policyGenerator.yaml (hunk 1 above)
Comment (gparvin) [inline policygenerator/policy-sets/community/zts-xcrypt/policyGenerator.yaml:42]: Add a more detailed description

File: policygenerator/policy-sets/community/zts-xcrypt/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+   % oc get nodes|grep worker
+   ip-10-0-143-90.us-east-2.compute.internal    Ready      worker   110d   v1.19.0+e49167a
+   ip-10-0-190-63.us-east-2.compute.internal    Ready      worker   110d   v1.19.0+e49167a
+   ip-10-0-207-48.us-east-2.compute.internal    Ready      worker   110d   v1.19.0+e49167a
+
+   Apply labels to nodes:
+   $ oc label nodes ip-10-0-143-90.us-east-2.compute.internal zts-master=true
+   $ oc label nodes ip-10-0-190-63.us-east-2.compute.internal zts-worker=true
+   $ oc label nodes ip-10-0-207-48.us-east-2.compute.internal zts-worker=true
```
Comment (gparvin) [inline policygenerator/policy-sets/community/zts-xcrypt/README.md:21]: Make this a code block

**Review by gparvin — COMMENTED** [review]
You need to sign off on your commits.  See https://github.com/stolostron/policy-collection/blob/main/CONTRIBUTING.md#dco-sign-off for details that I think will help.  Thanks for making this into a policy set.  I only had a few other comments about formatting the readme that you could optionally improve on.  Thanks

File: policygenerator/policy-sets/community/zts-xcrypt/policyGenerator.yaml (hunk 1 above)
Comment (mahesh-zetta) [reply policygenerator/policy-sets/community/zts-xcrypt/policyGenerator.yaml:42]: I created a new PR 366 which addresses the comments.

File: policygenerator/policy-sets/community/zts-xcrypt/README.md (hunk 2 above)
Comment (mahesh-zetta) [reply policygenerator/policy-sets/community/zts-xcrypt/README.md:47]: I created a new PR 366 which addresses the comments.

File: policygenerator/policy-sets/community/zts-xcrypt/README.md (hunk 3 above)
Comment (mahesh-zetta) [reply policygenerator/policy-sets/community/zts-xcrypt/README.md:21]: I created a new PR 366 which addresses the comments.

**Pull request closed without merging**